arviz/plots/elpdplot.py

```diff
@@ -144,7 +144,10 @@ def plot_elpd(
 
     if plot_kwargs is None:
         plot_kwargs = {}
-    plot_kwargs.setdefault("marker", "+")
+    if backend == "matplotlib":
+        plot_kwargs.setdefault("marker", "+")
```

Comment:
Adding on the other comment, if we want rcParams to control the marker in matplotlib elpd plot, we can just remove this line. That will make the marker fall back to its default in [matplotlib rcparams](https://github.com/matplotlib/matplotlib/blob/master/matplotlibrc.template#L603) and users will be able to modify it from there.

Reply:
The matplotlib rcParam has `.` as default marker option. This will make the matplotlib example and bokeh example generated graph different. Do we also need to modify the example code?

Reply:
No need to modify the example if you don't want to, it will use a circle instead of a cross.

Reply:
As in all other examples, both 'bokeh' and 'matplotlib' have the same graphs. I think it'll be better to add `plot_kwargs = {"marker":"+"}` to matplotlib files. It'll also be helpful to illustrate the use of `marker` in the plots. What are your thoughts?

Reply:
I'll start to add these changes to other scatter plots.

Reply:
I am not sure there are many instances of scatter plots, I think we generally use `plot` with linestyle `None`. The docs in elpdplot should be fixed though, `plot_kwargs` are passed to `plt.scatter` in this case

Reply:
@OriolAbril  Apart from scatter plots , normal `plot`'s marker can also be changed. For `essplot`, we are passing kwargs `{'color': 'red', 'linestyle': 'none', 'linewidth': 1.5, 'markersize': 6.0, 'marker': '+', 'zorder': 3, 'label': 'quantile'}` to `ax.plot`. We can change marker in kwarg to change the marker

Reply:
There is similar case for `mcseplot` where we are explicitly defining the kwargs.

Reply:
> I am not sure there are many instances of scatter plots, I think we generally use `plot` with linestyle `None`. The docs in elpdplot should be fixed though, `plot_kwargs` are passed to `plt.scatter` in this case

I have corrected the docstring.